src/custom-types.json

```diff
@@ -50,6 +50,20 @@
       "http://www.iana.org/assignments/media-types/application/geo+json"
     ]
   },
+  "application/grpc": {
+    "sources": [
+      "https://tools.ietf.org/html/draft-kumar-rtgwg-grpc-protocol-00",
+      "https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-HTTP2.md"
+    ]
+  },
+  "application/grpc+proto": {
+    "extensions": ["proto"],
```

Comment:
I tried to look through the three docs, but I didn't see anything that says that `.proto` == `application/grpc+proto`. Can you let me know in which doc and where that is located?